docs/gettingstarted/manifest-deployer-tutorial/README.md

```diff
@@ -0,0 +1,358 @@
+  version: v0.1.0
+meta:
+  schemaVersion: v2
+```
+
+**resources.yaml**: The resource.yaml contains a list of resource definitions for e.g. OCI images and Blueprints. The resources needed by our application are an OCI image containing the sample application and a blueprint describing the Landscaper deployment. Have a look at the resources and their yaml definition in /manifest-deployer-tutorial/resources.yaml. Later, it is shown how to update the component-descriptor.yaml with the resources specified in resources.yaml using the component-cli tool.
+
+**[blueprint](https://github.com/gardener/landscaper/blob/master/docs/concepts/Glossary.md#blueprint)**: This folder contains the blueprint.yaml. The Blueprint specifies any import parameters necessary for a Landscaper deployment, it specifies what kind of deployment should be executed (deploy-execution.yaml), and it specifies if the deployment creates output via export parameters.
+
+The application itself is not really of interest, it just keeps a container up and running. The container runs in a Pod, the  Pods has 3 Replicas, and all of this is specified in a Deployment manifest which you find in /manifest-deployer-tutorial/manifests/my-deployment.yaml. The other resource manifest we need for the application, is a Kubernetes resource of type Secret. This will allow Kubernetes to access the OCI registry which we deployed in the step "Install the Landscaper together with an OCI registry".
+
+Let us go through the steps, to prepare our Landscaper manifest deployment:
+
+1. In the component-descriptor.yaml (/manifest-deployer-tutorial/component-archive/), replace the `<OCIURL>` placeholder with the OCI registry URL as obtained in section "Install the Landscaper together with an OCI registry". E.g. replace `<OCIURL>` with o.ingress.p.democlust.shoot.k8s.myprovider.com.
+
+2. In the resources.yaml (/manifest-deployer-tutorial/component-archive/), replace the `<IMAGEURL>` placeholder with the container image URL used in section "Push demo image content into OCI registry", to push the sample image into our OCI registry. E.g. replace `<IMAGEURL>` with o.ingress.p.democlust.shoot.k8s.myprovider.com/hello.
+
+3. In deploy-excution.yaml (/manifest-deployer-tutorial/component-archive/blueprint/), replace the `<DOCKERCONFIG>` placeholder with a base64 encoded version of dockers config.json. You already did a docker login into your OCI registry, right?
+
+    ``` bash
+    cat ~/.docker/config.json | base64 --wrap=0
+    ```
+
+4. Add the resources defined in resource.yaml to component-descriptor.yaml by executing
+
+    ``` bash
+    component-cli component-archive resources add ./component-archive ./component-archive/resources.yaml
+    ```
+
+5. Verify updated content of component-descriptor. You need to replace the placeholder `<OCIURL>` with the OCI registry URL obtained in the section "Install the Landscaper together with an OCI registry", e.g. "o.ingress.p.democlust.shoot.k8s.myprovider.com"
```

Comment:
I'm missing some motivation about what we are doing here. Adding "o.ingress.p.democlust.shoot.k8s.myprovider.com" means that we defined the upload location of the component descriptor "o.ingress.p.democlust.shoot.k8s.myprovider.com/component-descriptors" in the OCI registry